### ISSUE TITLE
How to implement few-shot learning

### ISSUE THREAD

**Issue by L-JunJie** [LinglanZhao/BiDistFSCIL#5]
How to implement few-shot learning

**Comment by LinglanZhao** [comment]
Our method is proposed for few-shot class-incremental learning. For standard few-shot learning, I think you can only use the logits for novel classes for prediction.
